### ISSUE TITLE
Issue with latest wtforms release

### ISSUE THREAD

**Issue by 13129** [aminalaee/sqladmin#645]
### Checklist

- [X] The bug is reproducible against the latest release or `master`.
- [X] There are no similar issues or pull requests to fix it yet.

### In the admin click the Create or Modify button to get an error
```
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\uvicorn\protocols\http\h11_impl.py", line 408, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\uvicorn\middleware\proxy_headers.py", line 84, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\fastapi\applications.py", line 292, in __call__
    await super().__call__(scope, receive, send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\applications.py", line 122, in __call__
    await self.middleware_stack(scope, receive, send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\middleware\errors.py", line 184, in __call__
    raise exc
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\middleware\errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\middleware\cors.py", line 83, in __call__
    await self.app(scope, receive, send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\middleware\exceptions.py", line 79, in __call__
    raise exc
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\middleware\exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\fastapi\middleware\asyncexitstack.py", line 20, in __call__
    raise e
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\fastapi\middleware\asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\routing.py", line 443, in handle
    await self.app(scope, receive, send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\applications.py", line 122, in __call__
    await self.middleware_stack(scope, receive, send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\middleware\errors.py", line 184, in __call__
    raise exc
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\middleware\errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\middleware\exceptions.py", line 79, in __call__
    raise exc
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\middleware\exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\routing.py", line 66, in app
    response = await func(request)
               ^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\sqladmin\authentication.py", line 66, in wrapper_decorator
    return await func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\sqladmin\application.py", line 510, in create
    return self.templates.TemplateResponse(model_view.create_template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\templating.py", line 113, in TemplateResponse
    return _TemplateResponse(
           ^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\starlette\templating.py", line 39, in __init__
    content = template.render(context)
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\jinja2\environment.py", line 1301, in render
    self.environment.handle_exception()
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\sqladmin\templates\create.html", line 1, in top-level template code
    {% extends "layout.html" %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\sqladmin\templates\layout.html", line 2, in top-level template code
    {% from '_macros.html' import display_menu %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\sqladmin\templates\base.html", line 16, in top-level template code
    {% block body %}
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\sqladmin\templates\layout.html", line 49, in block 'body'
    {% block content %} {% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\sqladmin\templates\create.html", line 19, in block 'content'
    {{ field() }}
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\wtforms\fields\core.py", line 176, in __call__
    return self.meta.render_field(self, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\wtforms\meta.py", line 64, in render_field
    return field.widget(field, **render_kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\Workspace\py_frame\x_one_tool\venv\Lib\site-packages\wtforms\widgets\core.py", line 365, in __call__
    for val, label, selected, render_kw in field.iter_choices():
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: not enough values to unpack (expected 4, got 3)

```

### Debugging material

models.py
```
from sqlalchemy import Boolean
from sqlalchemy import Column
from sqlalchemy import String

from apps.base import BaseModel


# @as_declarative()
class DbTestModel(BaseModel):
    __tablename__ = 'db_test'
    # id = Column(String(64), primary_key=True, autoincrement=True)
    type_name = Column("type_name", String(50), comment="数据源类型名称")
    connect_string_default = Column("connect_string_default", String(500), comment="默认连接字符串")
    delete_status = Column("delete_status", Boolean, comment="默认连接字符串", default='False', nullable=True)
```
main.py
```
import os

import uvicorn
from fastapi import FastAPI
from fastapi.openapi.docs import (
    get_redoc_html,
    get_swagger_ui_html,
    get_swagger_ui_oauth2_redirect_html,
)
from fastapi.openapi.utils import get_openapi
from fastapi.staticfiles import StaticFiles
from starlette.middleware.cors import CORSMiddleware
from apps.db_catlog import db_c_api as db_c_api
from setting import settings

os.environ["TZ"] = settings.TIMEZONE
app = FastAPI(
    title=settings.TITLE,
    description=settings.DESCRIPTION,
    debug=settings.DEBUG,
    # log_level="info",
    docs_url=None,
    redoc_url=None,
    # openapi_url=None,
)

app.add_middleware(
    CORSMiddleware,
    allow_origins=["*"],
    allow_credentials=True,
    allow_methods=["*"],
    allow_headers=["*"],
)
app.include_router(db_c_api, prefix=settings.API_V1_STR)

from sqladmin import Admin, ModelView
from apps.test.models import DbTestModel
from apps.db import async_session

admin = Admin(app, session_maker=async_session, debug=True)


class UserAdmin(ModelView, model=DbTestModel):
    name = "User"
    name_plural = "Users"
    icon = "fa-solid fa-user"
    category = "accounts"

    can_view_details = True
    can_edit = True
    column_list = ["type_name"]
    column_details_list = ["type_name"]



admin.add_model_view(UserAdmin)
if __name__ == '__main__':
    uvicorn.run(app='main:app', reload=True, workers=1)
```

### Dependencies used
```
aiosqlite==0.19.0
alembic==1.12.0
annotated-types==0.6.0
anyio==3.7.1
click==8.1.7
colorama==0.4.6
fastapi==0.103.2
greenlet==3.0.0
h11==0.14.0
idna==3.4
itsdangerous==2.1.2
Jinja2==3.1.2
Mako==1.2.4
MarkupSafe==2.1.3
pydantic==2.4.2
pydantic-core==2.10.1
python-multipart==0.0.6
sniffio==1.3.0
sqladmin==0.15.1
SQLAlchemy==2.0.21
starlette==0.27.0
typing-extensions==4.8.0
uvicorn==0.23.2
wtforms==3.1.0

````

**Comment by aminalaee** [comment]
Yes, I saw this the other day as they have released a breaking change.I will work on a release soon.

**Comment by aminalaee** [comment]
Closed by #646
